modules/ingester/flush.go

```diff
@@ -32,10 +36,15 @@ var (
 	})
 )
 
+type opKind int
+
 const (
 	// Backoff for retrying 'immediate' flushes. Only counts for queue
 	// position, not wallclock time.
 	flushBackoff = 1 * time.Second
+
+	complete opKind = iota
```

Comment:
this will set both complete and flush to the same value i think.  i'd prefer a const block only for opkinds and it to look something like:
```
const (
opKindComplete = iota
opKindFlush
)
```

Reply:
They both get different values in the different version as well :) But the rewrite makes it clearer, updated.